doc/_sphinx/scripts/menu-expand.js

```diff
@@ -0,0 +1,69 @@
+// Auto expand the first expandable node ("flame") when loaded.
+window.addEventListener('load', (_event) => {
+    // expand first node when on home page
+    expandFirstOnHome();
+});
+
+/**
+ * This method expands the first expandable node on the home page.
+ * 
+ * If the current page is not the home page, this method does nothing.
+ */
+// Two scenarios are possible here:
+// 1. docs.flame-engine.org/ host
+//  - the pathname for homepage can be /{version}/index.html or /{version}/
+// 2. local host
+// - the pathname for homepage will end with /html/index.html
+function expandFirstOnHome() {
+    const parts = location.pathname.split('/');
+    const partsLength = parts.length;
+    let canExpandFirst = false;
+    if (location.host === 'docs.flame-engine.org') {
+        if (partsLength == 3 && parts[0] === '' && (parts[2] === '' || parts[2] === 'index.html')) {
+            canExpandFirst = true;
+        }
+    } else {
+        // local
+        if (parts[partsLength-2] == 'html' && parts[partsLength-1] == 'index.html') {
+            canExpandFirst = true;
+        }
+    }
```

Comment:
Since we have only one `index.html` page on the site, this can be simplified to a condition that is independent from the domain:
```suggestion
    const lastPart = parts[parts.length - 1];
    const isHomePage = (lastPage == '') || (lastPage == 'index.html');
```

Reply:
I initially thought this might not work when the lastPart is `''`. But, seems like whenever there is a blank lastPart (eg. https://docs.flame-engine.org/1.2.0/flame_audio) it goes to the home page of the latest version. So, I guess this should be working fine.